content/en/docs/security/otel-collector-configuration-best-practices.md

```diff
@@ -0,0 +1,147 @@
+processing the data in some way. From a security perspective, they are useful in
+a few ways.
+
+### Recommended Processors
+
+<!--- TODO: SHOULD configure recommended processors. If so, what are they? -->
+
+### Scrubbing sensitive data
+
+It is common for an OpenTelemetry Collector to be used to scrub sensitive data
+before exporting it to a backend. This is especially important when sending the
+data to a third-party backend. The OpenTelemetry Collector **SHOULD** be
+configured to obfuscate or scrub sensitive data before exporting.
+
+<!--- TODO: SHOULD configure obfuscation/scrubbing of sensitive metadata. How? Give more details and/or link to an existing document -->
+
+Use OpenTelemetry Collector's `redaction` processor to scrub sensitive data.
+
+### Safeguarding around resource utilization
+
+Processors also offer safeguards around resource utilization. After implementing
+[safeguards around resource utilization in your hosting](/security/otel-collector-hosting-best-practices/),
+ensure your OpenTelemetry Collector configuration uses these safeguards.
+
+<!-- start same page content in otel-collector-hosting-best-practices -->
+
+The `batch` and especially `memory_limiter` processor help ensure that the
+OpenTelemetry Collector is resource efficient and does not out of memory when
+overloaded. At least these two processors SHOULD be enabled on every defined
+pipeline.
+
+For more information on recommended processors and order, refer to the
+[OpenTelemetry Collector Processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor)
+documentation.
+
+<!-- /end same page content in otel-collector-hosting-best-practices -->
+
+## Extensions
+
+While receivers, processors, and exporters handle telemetry data directly,
+extensions typical serve different needs.
+
+<!--- TODO: Extensions SHOULD NOT expose sensitive health or telemetry data. How? What can you do? -->
+
+### Health and Telemetry
+
+The initial extensions provided health check information, Collector metrics and
+traces, and the ability to generate and collect profiling data. When enabled
+with their default settings, all of these extensions except the health check
+extension are only accessibly locally to the OpenTelemetry Collector. Care
+should be taken when configuring these extensions for remote access as sensitive
+information may be exposed as a result.
+
+### Collector's Internal Telemetry
+
+<!--- INSERT RECOMMENDATIONS HERE. For example:
+
+1. Remove zPages.
+1. Remove configuration endpoints.
+-->
+
+### Forwarding [component developers?]
+
+A forwarding extension is typically used when some telemetry data not natively
+supported by the OpenTelemetry Collector needs to be collected. For example, the
+`http_forwarder` extension can receive and forward HTTP payloads. Forwarding
+extensions are similar to receivers and exporters so the same security
+considerations apply.
+
+### Observers [component developers?]
```

Comment:
This is for end-users. Perhaps an example would be useful here, for users to understand what they can do with observers.

Here's one such example, where a redis receiver is created if a redis pod is found on a Kubernetes cluster:

https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/observer/k8sobserver

cc @rmfitzpatrick , @dmitryax

Reply:
Amended the header here, since Observers are for end-users. 👍 

An example here would be good to have. Still need to incorporate what you shared, @jpkrohling, but I appreciate the resource you shared!